cxx-qt-gen/test_outputs/basic_only_signals.cpp

```diff
@@ -0,0 +1,41 @@
+#include "cxx-qt-gen/include/my_object.h"
+#include "cxx-qt-gen/src/my_object.rs.h"
+
+namespace cxx_qt::my_object {
+
+MyObject::MyObject(QObject* parent)
+  : CxxQObject(parent)
+  , m_rustObj(createRs())
+{
+  initialiseCpp(*this);
+  m_initialised = true;
+}
+
+MyObject::~MyObject() = default;
+
+void
+MyObject::invokable()
+{
+  const std::lock_guard<std::mutex> guard(m_rustObjMutex);
+  m_rustObj->invokableWrapper(*this);
+}
+
+void
+MyObject::emitReady()
+{
+  runOnGUIThread([=]() { Q_EMIT ready(); });
```

Comment:
We could instead use `QMetaObject::invokeMethod(this, "ready", Qt::QueueConnection)` which with possible changes in #12 could allow for removing `runOnGUIThread` etc if UpdateRequester also used this route.